MapboxNavigation/NavigationViewController.swift

```diff
@@ -409,7 +409,7 @@ public class NavigationViewController: UIViewController {
 
         mapViewController?.notifyDidChange(routeProgress: routeProgress, location: location, secondsRemaining: secondsRemaining)
 
-        if automaticallySwitchStyleForTunnels,
+        if enableNightStyleInsideTunnelFeatureEnabled,
```

Comment:
This property has the word `enable(d)` in it twice. I too am a fan of suggestion - `automaticallySwitchesStyleInsideTunnels`.

Reply:
Per our discussion, this variable will be `usesNightStyleInsideTunnels`.

cc @bsudekum @1ec5